package.json

```diff
@@ -34,6 +34,7 @@
     "@player.style/microvideo": "0.0.5",
     "@player.style/minimal": "0.0.5",
     "@player.style/yt": "0.0.5",
+    "hls.js": "^1.5.13",
```

Comment:
this shouldn't be needed, did you run `npm install` in the root of this monorepo?

Reply:
Pretty sure? I don't think I ever did anything outside of the root of the monorepo, but I'll try a clean install this afternoon and see if I can reproduce.